### ISSUE TITLE
Set up some kind of chat service

### ISSUE THREAD

**Issue by lucaswoj** [manifold-lang/manifold#156]

**Comment by mtrberzi** [comment]
We had this with Flowdock, but...no one uses that any more :(

**Comment by lucaswoj** [comment]
Correct. We should get something like that up again :+1: Flowdock wanted money eventually

**Comment by mtrberzi** [comment]
I don't know how useful this would be given that we now have "third-party" contributors. Maybe some kind of mailing list would be better.
